pkg/registry/event/strategy_test.go

```diff
@@ -84,3 +86,12 @@ func TestGetAttrs(t *testing.T) {
 		t.Errorf("diff: %s", util.ObjectDiff(e, a))
 	}
 }
+
+func TestSelectableFieldLabelConversions(t *testing.T) {
+	apitesting.TestSelectableFieldLabelConversionsOfKind(t,
+		testapi.Default.GroupVersion().String(),
+		"Event",
+		labels.Set(generic.ObjectMetaFieldsSet(api.Event{}.ObjectMeta, true)),
```

Comment:
Why not using the getAttrs() method defined in the strategy? `Event` supports fields other than what the `ObjectMetaFieldsSet` returns.